doc/examples/auto_csfle_example.py

```diff
@@ -0,0 +1,105 @@
+import os
+import asyncio
+
+from bson.codec_options import CodecOptions
+from bson import json_util
+
+from motor.motor_asyncio import AsyncIOMotorClient
+from pymongo.encryption import (Algorithm,
+                                ClientEncryption)
+from pymongo.encryption_options import AutoEncryptionOpts
+
+
+def create_json_schema_file(kms_providers, key_vault_namespace,
+                            key_vault_client):
+    client_encryption = ClientEncryption(
```

Comment:
So is the plan to add these examples now and then update them once we implement the async motor ClientEncryption wrapper class?

Reply:
yep! For automatic CSFLE this should work fine, but we'll use the wrapper once I figure it out